### ISSUE TITLE
improved keyboard navigation for the New Issue form

### ISSUE THREAD

**Issue by maxcan** [waffleio/waffle.io#2417]
there should be a way to fully keyboard through the new issue dialog. unfortuantely, the repo select dropdown requires using the mouse to choose the repo.  I've tried hitting down arrow or just typing hoping for a typeahead experience

**Comment by brianpeiris** [comment]
Keyboard navigation is also missing on the "Select Labels" field.
